pennylane/devices/default_qubit.py

```diff
@@ -527,19 +525,23 @@ def _apply_basis_state(self, state, wires):
 
         self._state = self._create_basis_state(num)
 
-    def _apply_unitary(self, mat, wires):
+    def _apply_unitary(self, state, mat, wires):
```

Comment:
Ideally we could think about making these as static methods or simply functions, but we do still use `self` for things like `self._tensordot`, i.e., interface-dependent calculations.

Reply:
I second making them static functions 👍 
Keeping them methods in the class keeps them organized and contained a bit better, but they should probably be made static methods.

Reply:
This is a nice idea, which would make things even cleaner. :+1:

Reply:
Note: eventually `self._tensordot` and all the other ML static methods can be replaced by `qml.math.tensordot` etc.

But until we do that, we'll likely need to keep them as instance methods 🙁

Reply:
Oh awesome, am loving the idea to use `qml.math`!